include/usb_cam/usb_cam.hpp

```diff
@@ -314,6 +320,79 @@ class UsbCam
     return m_image.pixel_format;
   }
 
+  /// @brief Send current parameters to V4L2 device
+  /// TODO(flynneva): only send parameters that changed
+  inline void set_v4l2_params()
```

Comment:
Are the inline functions really needed here?  Maybe, it is better to implement the classic prototype + body model? For the function called only once it is not ever needed, why we should keep this [useless] specifier here? I guess, it is inherited from the legacy and we can remove the `inline` specifiers without any problem. This is only my opinion, so please share what you think about any real reason we have here